src/main/java/com/icthh/xm/ms/mstemplate/web/rest/utils/RespContentUtil.java

```diff
@@ -0,0 +1,52 @@
+package com.icthh.xm.ms.mstemplate.web.rest.utils;
+
+import com.icthh.xm.commons.exceptions.EntityNotFoundException;
+import java.util.Optional;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+public final class RespContentUtil {
```

Comment:
Questions regarding naming:

1. Why not full name: Response, looks like other utilities class has full naming
2. Why not plural?: for example: TenantContextUtils, StringUtils etc

Reply:
fixed